osu.Game/Skinning/LegacyBeatmapSkin.cs

```diff
@@ -45,11 +45,11 @@ private static IResourceStore<byte[]> createRealmBackedStore(BeatmapInfo beatmap
 
         public override Drawable? GetDrawableComponent(ISkinComponentLookup lookup)
         {
-            if (lookup is GlobalSkinComponentLookup targetComponent)
+            if (lookup is SkinComponentsContainerLookup targetComponent)
```

Comment:
Old var name (another place changed it to `componentLookup`, but I'd probably go even further for `containerLookup` or `componentsContainerLookup`). Former name seems inappropriate (because it's not a single component that's' being looked up).